CONTRIBUTING.md

```diff
@@ -11,50 +11,46 @@ To contribute code changes or update the documentation, please follow these step
 5. Create a pull request to the main repository with an explanation of your changes. The PR should detail the
    contribution and link to any related issues - if existing.
 
-## Code Contribution Guidelines
+## Code contribution guidelines
```

Comment:
I could be nitpicky about this but shouldn't the first letter of each word be capitalised? It can make the sentence stand out distinctively.

Down the line, I think a standardised writing style should be followed. Here's a great article discussing the many standard writing styles - [What Words Should be Capitalised in a Headline?]( https://bettermarketing.pub/what-words-should-be-capitalized-in-a-headline-1cd311f617b5).

Reply:
This currently varies quite a lot across our docs, and it something that should be addressed. Personally, I tend to just follow proper capitalisation. I just the article and the takeaway seems to be that there's no "correct way" of doing it so, let's just decide on one and go with it?

Reply:
I would vote for Pascal-case type capitalization for header-type sections.
I think it looks cleaner even if it is not proper grammar.

Some of our current  examples: 
https://github.com/starlite-api/starlite#dependency-injection
https://github.com/starlite-api/starlite#route-guards
https://github.com/starlite-api/starlite#request-life-cycle-hooks

Reply:
I follow proper capitalised words for the headers as well for documentations, so my vote is on that boat instead.

Reply:
Let's open a separate discussion about these topics.

Reply:
@JacobCoffee, @Jarmos-san: #1092